packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerContent.tsx

```diff
@@ -292,7 +294,8 @@ const getStyles = (
     flexDirection: 'column',
     borderRight: `${isReversed ? 'none' : `1px solid ${theme.colors.border.weak}`}`,
     width: `${!hideQuickRanges ? '60%' : '100%'}`,
-    overflow: 'hidden',
+    overflowY: 'scroll',
+    overflowX: 'hidden',
```

Comment:
you can't really mix these. by setting `overflowY: scroll`, `overflowX` is implicitly set as well (see https://stackoverflow.com/a/60538603). to avoid confusion, i'd just set `overflow: auto`.

Reply:
I didn't know that I couldn't mix them. Thanks for the info.